### DISCUSSION TITLE
【灌水】我很好奇大家有没有去过匹诺康尼？

### DISCUSSION THREAD

**Discussion by AwA-114514-awa** [share121/inter-knot#2000] [其他]
![R-C](https://github.com/user-attachments/assets/c1b24f92-7568-4d7e-b858-51d17d213912)
我是来自公司的一名基层员工，去皮诺康尼执行任务，当了当安保，在当安保的时候遇到了一个令我难忘的人
一个灰色头发人，在贝洛伯格把我打了一顿，听说是很厉害的人，害得我差点业绩都扣光了  awa

**Comment by YING-ADX** [comment]
a？6
